catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -72,6 +81,8 @@
 
     private Integer subsampleCount = DEFAULT_SUBSAMPLE_COUNT;
 
+    private Function<MpegStreamType, String> streamTypeToString = new StreamTypeToString();
```

Comment:
is there any reason this isn't just
```java
private Function<MpegStreamType, String> streamTypeToString = (streamType) -> streamType.toString()
```

I don't think the StreamTypeToString class is needed